Shared/mods/deathmatch/logic/luadefs/CLuaUtilDefs.cpp

```diff
@@ -190,7 +190,8 @@ int CLuaUtilDefs::Split(lua_State* luaVM)
         char*        szToken = strtok(strText, strDelimiter);
 
         // Create a new table
-        lua_newtable(luaVM);
+        // Reserve a generous amount, because it'll get GCd fast probably
```

Comment:
This isn't particularly generous.

```suggestion
        // Reserve a small amount
```

Reply:
Well, considering a split is usually a few tokens :D